config/notifiers.go

```diff
@@ -491,6 +492,10 @@ func (c *OpsGenieConfig) UnmarshalYAML(unmarshal func(interface{}) error) error
 		}
 	}
 
+	if c.Details != nil && c.DetailsUseCommonLabels == true {
+		return errors.Errorf("OpsGenieConfig can only contain details or details_use_common_labels, but both fields were provided")
```

Comment:
I have a tiny proposal: What about having this option `CommonLabelsAsDetails` and if true it will populate details from common labels, and then anything in `Details` will be added **additionally**? It will be bit more flexible for user and also quite simpler to implement and ensure. (: